src/messages/Image.cpp

```diff
@@ -455,6 +455,16 @@ int Image::height() const
         return 16;
 }
 
+QSize Image::size() const
+{
+    assertInGuiThread();
+
+    if (auto pixmap = this->frames_->first())
+        return pixmap->size() * this->scale_;
+    else
+        return {16, 16};
```

Comment:
7TV and FFZ include the size of the image/emote when loading emotes - we might be able to use this here (or this could be added in a followup PR).

Reply:
I added a comment in the code for this :+1: